Sql_Assignment

```diff
@@ -0,0 +1,310 @@
+
+
+3). List the OrderHeader table contents so that the latest placed order appears in the top.
+
+select * from Order_Header order by Order_Id desc;
+
++----------+---------------+--------------+-------------------------+----------------+-----------------+-------------------+-----------------------+-------------+
+| ORDER_ID | ORDER_TYPE_ID | ORDER_NAME   | ORDER_DATE              | STATUS_ID      | CURRENCY_UOM_ID | PRODUCT_STORE_ID  | SALES_CHANNEL_ENUM_ID | GRAND_TOTAL |
++----------+---------------+--------------+-------------------------+----------------+-----------------+-------------------+-----------------------+-------------+
+| 1255     | SALES_ORDER   | Test Order 5 | 2020-03-17 00:00:00.000 | ORDER_APPROVED | USD             | OMS_DEFAULT_STORE | WEB_SALES_CHANNEL     |     75.9300 |
+| 1153     | SALES_ORDER   | Test Order 4 | 2020-04-19 00:00:00.000 | ORDER_APPROVED | USD             | OMS_DEFAULT_STORE | WEB_SALES_CHANNEL     |     31.6400 |
+| 1102     | SALES_ORDER   | Test Order 3 | 2020-04-17 00:00:00.000 | ORDER_APPROVED | USD             | OMS_DEFAULT_STORE | WEB_SALES_CHANNEL     |    129.9200 |
+| 1064     | SALES_ORDER   | Test Order 7 | 2022-01-09 00:00:00.000 | ORDER_APPROVED | USD             | OMS_DEFAULT_STORE | WEB_SALES_CHANNEL     |     48.8700 |
+| 1061     | SALES_ORDER   | Test Order 6 | 2019-04-24 00:00:00.000 | ORDER_APPROVED | USD             | OMS_DEFAULT_STORE | WEB_SALES_CHANNEL     |     87.0700 |
+| 1052     | SALES_ORDER   | Test Order 2 | 2020-04-17 00:00:00.000 | ORDER_APPROVED | USD             | OMS_DEFAULT_STORE | WEB_SALES_CHANNEL     |     16.3200 |
+| 1051     | SALES_ORDER   | Test Order 1 | 2020-04-17 00:00:00.000 | ORDER_APPROVED | USD             | OMS_DEFAULT_STORE | WEB_SALES_CHANNEL     |     77.9500 |
++----------+---------------+--------------+-------------------------+----------------+-----------------+-------------------+-----------------------+--
+
+4).Output the count of OrderItem Ship Group for each shipmentMethodTypeId.
+
+select Count(ship_group_seq_id) AS count_of_ship_group from order_item_ship_group group by shipment_method_type_Id;
++---------------------+
+| count_of_ship_group |
++---------------------+
+|                  12 |
++---------------------+
+
+5). List the OrderHeader details with grandTotal greater than 50.
+
+select Grand_Total from Order_Header where Grand_Total >= 50;
++-------------+
+| grand_total |
++-------------+
+|     77.9500 |
+|     87.0700 |
+|    129.9200 |
+|     75.9300 |
++-------------+
+
+6). List the Person details who have not placed any orders.
+
+select Party_Id, First_Name from Person where Party_Id NOT IN (Select Customer_Party_Id from Order_Item_Ship_Group);
++-----------+------------+
+| Party_Id  | First_Name |
++-----------+------------+
+| CustDemo2 | Jack       |
++-----------+------------+
+
+
+7).List the Person details who have not placed any orders.
```

Comment:
The query doesn't go with this question.

Reply:
07).List the Person details whose first name starts with ‘J’ and third character is ’h’.